src/pageEditor/Editor.tsx

```
@@ -175,18 +176,22 @@ const Editor: React.FunctionComponent = () => {
   }
 
   return (
-    <div className={styles.root}>
-      <Sidebar
-        installed={installed}
-        elements={elements}
-        recipes={recipes}
-        activeElementId={activeElementId}
-        activeRecipeId={activeRecipeId}
-        isInsertingElement={Boolean(inserting)}
-        isLoadingItems={loadingRecipes}
-      />
-      {body}
-    </div>
+    <>
+      <div className={styles.root}>
+        <Sidebar
+          installed={installed}
+          elements={elements}
+          recipes={recipes}
+          activeElementId={activeElementId}
+          activeRecipeId={activeRecipeId}
+          isInsertingElement={Boolean(inserting)}
+          isLoadingItems={loadingRecipes}
+        />
+        {body}
+      </div>
+
+      <AddToRecipeModal />
```

NIT: I'd consider having this parent component use the selectIsShowingAddToRecipeModal component and conditionally render the modal. Then you wouldn't have to worry about reseting to defaults on hide